frontend/src/Components/ProfilePages/VolunteerProfilePage.jsx

```diff
@@ -19,6 +21,29 @@ export default function VolunteerProfilePage(props) {
                     setVolunteer(data.payload);
                     setProfilePublic(data.payload.v_id);
                     setWaitingForData(false);
+
+                    if (data.payload.mentees) {
+                        const parsedMentees = data.payload.mentees.map(mentee => mentee.split(' &$%& '));
+                        /* After splitting, for each mentee we will have:
+                            index0: mentee id
+                            index1: full name
+                            index2: When the mentoring relation started
+                            index3: Boolean: true relation ended, true it's still on
```

Comment:
I think you meant to write false for one of these:
Boolean: true relation ended, true it's still on